components/windows/manifesto.tsx

```diff
@@ -1,13 +1,62 @@
-import { useMemo } from "react";
-import { Box, Image, Link, Text, useMediaQuery } from "@chakra-ui/react";
-
+import {
+  Box,
+  Flex,
+  Icon,
+  Image,
+  Link,
+  type ResponsiveValue,
+  Text,
+  useMediaQuery
+} from "@chakra-ui/react";
+import { IoIosPlay, IoIosPause } from "react-icons/io";
 import { Window } from "@/components/window";
 import { useWindowManager } from "@/components/window-manager";
+import { useMemo } from "react";
+import { MotionBox } from "@/components/motion-box";
+import { useAudio } from "@/hooks/useAudio";
+
+const ManifestoSectionTitle = ({
+  title,
+  width
+}: {
+  title: string;
+  width: string;
+}) => (
+  <Flex alignItems={"center"} justifyContent={"center"} gap={6} mt={8} mb={2}>
+    <Image w={width} src="/decoration-frame.svg" />
+    <Text as="h2" mr={2} fontSize={{ base: "16px", sm: "22px" }}>
+      {title}
+    </Text>
+    <Image w={width} src="/decoration-frame.svg" />
+  </Flex>
+);
+
+const ManifestoBoldText = ({ text }: { text: string }) => (
+  <Text as="span" fontWeight={700}>
+    {text}
+  </Text>
+);
+
+const ManifestoLinkText = ({ text }: { text: string }) => (
+  <Link
+    href="https://openai.com/dall-e-2"
```

Comment:
is this correct link? why do we point dall e 2

Reply:
All the link are removed after Jesse asked for it